flink-autoscaler/src/test/java/org/apache/flink/autoscaler/ScalingTrackingTest.java

```diff
@@ -52,6 +52,7 @@ void setUp() {
         conf.set(AutoScalerOptions.PREFER_TRACKED_RESTART_TIME, true);
     }
 
+    // ------------- getMaxRestartTimeOrDefault -------------
```

Comment:
What does this comment mean?

Reply:
Why is it formatted that way?

Reply:
The test was growing large and I added sections for easier navigation. The name indicates the method that is the focus for the tests in the respective section. Formatting is arbitrary. We could do something like this, in line with how code blocks are sometimes demarcated in Flink:
```
    // ------------------------------------------------------------------------
    //  Tests for getMaxRestartTimeOrDefault()
    // ------------------------------------------------------------------------
```

Reply:
I would advise against these kind of comments. Similarly to JavaDocs, they tend to add more confusion than actually helping new people. Plus, they become outdated quickly.

Reply:
removed...